base_comment_template/views/res_partner_view.xml

```diff
@@ -6,14 +6,21 @@
             <page name='internal_notes' position="after">
                 <page name='base_comment_template_ids' string="Comment Templates">
                     <field
-                        colspan="4"
-                        mode="tree"
                         name="base_comment_template_ids"
                         nolabel="1"
-                        widget="one2many"
-                        height="300"
-                        context="{'default_partner_id': active_id}"
-                    />
+                        context="{'add_models_in_name': True}"
```

Comment:
Remember to change the context name here. Is this working putting the context at this level?